public/locales/fr/status.json

```diff
@@ -12,7 +12,7 @@
   "description": "Utilisez ce service avec votre numéro de dossier (FEDS) pour vérifier le statut de votre demande de passeport.",
   "esrf": {
     "error": {
-      "length": "Le numéro de dossier doit comporter 8 caractères.",
+      "length": "Le numéro de dossier peut comporter un maximum de 8 caractères.",
```

Comment:
I think this is better:

`Le numéro de dossier peut avoir un maximum de 8 caractères alphanumériques.`

Reply:
I'm okay with this given the recent introduction of letters into the ESRF, but given the original `length` string was provided to us by the BAs this might change.